src/components/ExpansionPanel/ExpansionPanel.tsx

```diff
@@ -45,20 +45,28 @@ export const useStyles = makeStyles(
     },
     leadingIcon: {
       marginRight: theme.spacing(1),
+      minWidth: 'fit-content',
     },
     titleContainer: {
       alignItems: 'center',
       display: 'flex',
-      width: '100%',
+      maxWidth: '95%',
```

Comment:
```suggestion
      maxWidth: `calc(100% - ${theme.pxToRem(34)})`,
```
Can we use `calc` here for more consistent padding between title and icon at varying browser widths?

I got `34` by taking `Plus` icon width `18px` + horizontal button padding `16px`

Reply:
On it!